tools/automation/README.md

```diff
@@ -0,0 +1,212 @@
+Open Model Zoo automation tools
+===============================
+
+This directory contains scripts that automate certain model-related tasks
+based on the included configuration file.
+
+* `downloader.py` (model downloader) downloads model files from the Internet
```

Comment:
`internet` -> `public sources`?

Reply:
Well... it wouldn't be incorrect, but is it better?

Reply:
I ended up replacing it with "online sources", which IMO combines the best of both wordings.